atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraKeyValueServiceConfig.java

```diff
@@ -128,6 +130,24 @@ default int gcGraceSeconds() {
         return CassandraConstants.DEFAULT_GC_GRACE_SECONDS;
     }
 
+    /**
+     * This increases the likelihood of selecting an instance that is hosted in the same data centre as the process.
+     * Weighting is a ratio from 0 to 1, where 0 disables the feature and 1 forces the same data centre if possible.
+     */
+    @Value.Default
+    default double localHostWeighting() {
+        return 0.0;
+    }
+
+    /**
+     * Controls the behaviour of the host location supplier when the snitch description does not match any known
+     * snitches.
+     */
+    @Value.Default
+    default Supplier<Optional<HostLocation>> defaultHostLocationSupplier() {
```

Comment:
Don't have this be a `Supplier`, just have it be the HostLocation. When you need a supplier, write `config::hostLocation` or similar.

Reply:
if you are writing methods which return suppliers but take no arguments and don't try to do anything clever, assume you're making a mistake, because the method reference itself can be thought of as the supplier.